libraries/AP_Logger/LogFile.cpp

```diff
@@ -1534,7 +1534,7 @@ void AP_Logger::Write_Airspeed(AP_Airspeed &airspeed)
             temperature   : (int16_t)(temperature * 100.0f),
             rawpressure   : airspeed.get_corrected_pressure(i),
             offset        : airspeed.get_offset(i),
-            use           : airspeed.use(i),
+            use           : (bool)airspeed.use(i),
```

Comment:
why cast to bool here?

Reply:
It has been returned back to returning a boolean so this is no longer required.  But I think there was an underlying issue in that I dont understand why I needed to cast it to a bool.  Surely it would have picked up the change in size automatically?  Or did I miss something there?